jdaviz/configs/default/plugins/data_quality/data_quality.py

```diff
@@ -0,0 +1,200 @@
+import os
+from traitlets import Any, Dict, Bool, List, Unicode, observe
+
+import numpy as np
+from glue_jupyter.common.toolbar_vuetify import read_icon
+from echo import delay_callback
+from matplotlib.colors import hex2color
+
+from jdaviz.core.registries import tray_registry
+from jdaviz.core.template_mixin import (
+    PluginTemplateMixin, ViewerSelect, LayerSelect
+)
+from jdaviz.core.tools import ICON_DIR
+from jdaviz.configs.default.plugins.data_quality.dq_utils import (
+    decode_flags, generate_listed_colormap, dq_flag_map_paths, load_flag_map
+)
+
+__all__ = ['DataQuality']
+
+telescope_names = {
+    "jwst": "JWST",
+    "roman": "Roman"
+}
+
+
+@tray_registry('g-data-quality', label="Data Quality", viewer_requirements="image")
+class DataQuality(PluginTemplateMixin):
+    template_file = __file__, "data_quality.vue"
+
+    # TODO: uncomment this line before merging into main:
+    # irrelevant_msg = Unicode("Data Quality plugin is in development.").tag(sync=True)
```

Comment:
you can always uncomment this and just reset it to empty in your own testing notebook and any tests 🤷‍♂️

Reply:
is the intention still to include this or is it mature enough that we can remove it entirely and have it enabled in main on merge (and whenever 3.10 is released)?

Reply:
(we could also have this be considered irrelevant if there are no data quality layers available... 🤔

Reply:
I’m in favor of keeping it out until we decide it’s more stable. It’s likely some things will change when we add support for other configs. Hopefully they aren’t major things, but just in case, let’s defer.